source/PlayerInfo.cpp

```diff
@@ -4884,6 +4884,38 @@ void PlayerInfo::DoAccounting()
 		message += ".";
 		Messages::Add(message, Messages::Importance::High, true);
 		accounts.AddCredits(salariesIncome + tributeIncome + balance.assetsReturns);
+
+		if(tributeIncome)
+		{
+			// Apply reputation penalties for dominated planets.
+			set<const Government *> governments;
+			for(const auto &it : tributeReceived)
+			{
+				if(it.first->DailyTributePenalty())
+				{
+					const Government *gov = it.first->GetGovernment();
+					gov->AddReputation(-it.first->DailyTributePenalty());
+					governments.insert(gov);
+				}
+			}
+			if(!governments.empty())
+			{
+				message = "You have lost reputation with the ";
+				// TODO: use Format::List
+				if(governments.size() == 1)
+					message += (*governments.cbegin())->GetName();
+				else
+					for(auto it = governments.cbegin(); it != governments.cend(); ++it)
+					{
+						if(it == prev(governments.cend()))
+							message += "and the " + (*it)->GetName();
+						else
+							message += (*it)->GetName() + ", ";
+					}
```

Comment:
If there are two items in the list, this will provude:
`the x, and the y`, which is not correct use of hte Oxford comma. It should be `the x and the y`.